core/src/db/entity/epub.rs

```diff
@@ -14,6 +14,7 @@ use super::{media::Media, MediaAnnotation};
 pub struct EpubContent {
 	label: String,
 	content: PathBuf,
+	children: Vec<EpubContent>,
```

Comment:
I'm surprised that clippy didn't complain about this recursive type, but I won't complain about it not complaining about it 😛